airbyte-integrations/connectors/source-airtable/spec.patch.json

```diff
@@ -0,0 +1,38 @@
+{
+      "properties": {
+        "credentials": {
+          "title": "Authentication",
+          "type": "object",
+          "discriminator": {
+            "propertyName": "auth_method"
+          },
+          "oneOf": [
```

Comment:
Arrays do not get merged by [JSON Merge Patch](https://www.rfc-editor.org/rfc/rfc7396.txt), and so *all* properties must be included when we are overwriting arrays. In this case we are missing `client_id`, `client_secret`, `refresh_token`, `api_key`, `access_token` and `token_expiry_date`

Reply:
@mdibaiee Is everything in the `oneOf` considered part of the array? If so, the `api_key` in the Personal Access Token section also is part of the `oneOf` array? Should I add the `api_key` section back in too?

Reply:
@jman0119 yes